index.ts

```diff
@@ -9,30 +9,64 @@ const moment = require('moment');
 // const io = require('socket.io').listen(http);
 const port = process.env.PORT || 5000;
 
-// const server = app()
-    // .use((req, res) => res.sendFile(INDEX, { root: __dirname }))
-    // .listen(port, () => console.log(`Listening on ${port}`));
+const {Client} = require('pg');
 
-const server = http.listen(port, (err) => {
-    if (err) throw err;
-    console.log("HTTP server listening on: " + port);
+const client = new Client({
+    connectionString: process.env.DATABASE_URL,
+    ssl: {
+        rejectUnauthorized: false
+    },
+    user: process.env.USER,
+    password: process.env.PASSWORD,
+    host: process.env.HOST,
+    database: process.env.DATABASE,
+});
+
+client.connect()
+    .catch((err) => {
+        console.error(err);
+        client.end();
+    });
+
+const knex = require('knex')({
+    client: 'pg',
+    version: '7.2',
+    connection: {
+        host: process.env.HOST,
+        user: process.env.USER,
+        password: process.env.PASSWORD,
+        database: process.env.DATABASE,
+        ssl: true
+    }
 });
 
-const { setWsHeartbeat } = require("ws-heartbeat/server");
-const { Server } = require('ws');
+// const server = app()
+// .use((req, res) => res.sendFile(INDEX, { root: __dirname }))
+// .listen(port, () => console.log(`Listening on ${port}`));
 
-const wss = new Server({ server });
+const {setWsHeartbeat} = require("ws-heartbeat/server");
+const {Server} = require('ws');
 
-class User {
-    constructor(public username: string, public usernum: number, public currentChannel: number, public id: number, public messages: Array<number>){}
+interface User {
+    username: string,
+    userNum: number,
```

Comment:
Idk, it kinda looks bad to have the n in username be lowercase and he one in useNum to be uppercase, but maybe it's just me